### PR TITLE
Redesigned the searchBar. Different look in Dark and Light ModeFeature

### LINKED ISSUE #37: Improve the `SearchView`
Currently the `SearchFragment` uses a basic search view which is not at all appealing, improve upon the design of the search view to make it similar to those present in most of the apps. Try to make the design relevant to its purpose, i.e., searching movies and TV shows

### PR DESCRIPTION
**Pull request by neil0504**
## Description

Before the search bar had default look but not it has been designed to a design similar to Instagram(not the functionality just the design)

## Issue Resolved

Fixed #37 

## Screenshot
Light Theme:
nl
s 

Dark Theme:
nd

## Checklist

Please make sure to review the following before submitting your PR: 

- [X] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [X] I have read the code of conduct.
- [X] I have reviewed my submission in detail.

### CODE REVIEW

File: app/src/main/res/layout/fragment_search.xml [hunk 1]
```diff
@@ -6,13 +6,21 @@
...
-    <androidx.appcompat.widget.SearchView
+    <SearchView
         android:id="@+id/search_text"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        app:layout_constraintTop_toTopOf="parent"
-        app:queryHint="Search movie" />
+        android:layout_height="35dp"
+        android:layout_margin="15dp"
+        android:layout_marginTop="40dp"
```
Comment (ken1000minus7) [inline app/src/main/res/layout/fragment_search.xml]: Pls make use of sdp values here instead of dp 😇

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]
Great work @neil0504 😄 ,  just a small change required, then your pr will be ready to be merged 🥳

File: app/src/main/res/layout/fragment_search.xml (hunk 1 above)
Comment (neil0504) [reply app/src/main/res/layout/fragment_search.xml]: Yeah, I have done the changes. Hope it solves the issues.

**Review by ken1000minus7 — APPROVED** [review]
Lgtm, great work ! @neil0504 😄

**Comment by neil0504** [comment]
> Lgtm, great work ! @neil0504 😄

Thank you @ken1000minus7 . it was great working on this PR

**Pull request merged**